include/tut/tut_assert.hpp

```diff
@@ -130,12 +130,16 @@ void ensure_equals(const M& msg, const LHS * const actual, const RHS * const exp
     }
 }
 
-template<typename M>
-void ensure_equals(const M& msg, const double& actual, const double& expected, const double& epsilon)
+template<typename M, typename T>
+typename std::enable_if<std::is_floating_point<T>::value, void>::type
+    ensure_equals(const M& msg, const T& actual, const T& expected, const T& epsilon)
```

Comment:
This is the C++11 version.

The C++14 version would be `std::is_floating_point<T>()`

The C++17 version would be `std::is_floating_point_v<T>`

Reply:
As far as I know, `std::is_floating_point<T>::value` is still valid and the two forms you mention are just syntactic sugar?

Reply:
I think you're missing `#include <type_traits>` before using `std::enable_if`. Also, this needs to be properly wrapped in `#if __cplusplus >= ...` clause or rewritten to work in C++98 mode.

Reply:
True, I use MSVC at work, which is quite permissive on STL includes...